src/Select/Select.less

```diff
@@ -0,0 +1,69 @@
+@import "../colors.less";
+
+.Select--container {
+  position: relative;
+
+  .Select--labelContainer {
+    color: @neutral_dark_gray;
+    font-size: 10px;
+    position: absolute;
+    text-transform: uppercase;
+    top: 5px;
+    width: 100%;
+    z-index: 1;
+
+    &.Select--labelHidden {
+      opacity: 0 !important;
+    }
+
+    .Select--label {
+      left: 10px;
+      position: absolute;
+    }
+  }
+
+  .Select--ReactSelect {
+    // overrides styles from the exact same selector in react-select's CSS
+    &.is-focused:not(.is-open) > .Select-control {
+      border-color: @neutral_silver;
+      box-shadow: inset 5px 0 @primary_blue;
+      transition: box-shadow .25s ease-out;
+    }
+
+    .Select-control {
+      border-radius: 0px;
+      border-color: @neutral_silver;
+      height: 50px;
+      .Select-placeholder {
+        font-size: 12px;
+        line-height: 40px;
+        padding-top: 5px;
+        text-transform: uppercase;
+      }
+      .Select-value {
+        font-size: 15px;
+        line-height: 40px;
+        min-height: 40px;
+        padding-top: 10px;
+      }
+    }
+
+    // overrides styles from the exact same selector in react-select's CSS
```

Comment:
I'm not too familiar with best practices overriding other selections (if there's any) -- would probably defer on @CarlNelson on this